### DISCUSSION TITLE
Frigate beta7 and Codeproject

### DISCUSSION THREAD

**Discussion by rodrigoratan** [blakeblackshear/frigate#9125] [Ask A Question]
from time to time frigate will report a huge inference speed from codeproject:
![image](https://github.com/blakeblackshear/frigate/assets/964066/9ca516b7-50d4-46c1-a336-fd2c847fcf57)

But codeproject is reporting ~50ms inference speeds
![image](https://github.com/blakeblackshear/frigate/assets/964066/86fe3bb8-f681-45af-9703-bcb43ca82ff7)

Does frigate crashes any internal service when using codeproject and stop working?

**Comment by NickM-27** [comment]
Codeproject.ai only reports the inference speed of itself, frigate inference time will also include the network latency too so they will not be the same. In some cases like this it would seem the request a timeout might've happened. 

I'd suggest checking the logs.
